qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -89,6 +89,27 @@ def _save_mount_details(self):
         self._orig_mount_details = [MountDetails(m) for m in self.mounts]
         log.info(self._orig_mount_details)
 
+    def _remove_blocklist(self):
+        # In case anything is in the OSD blocklist list, clear it out.  This is to avoid
+        # the OSD map changing in the background (due to blocklist expiry) while tests run.
+        try:
+            self.mds_cluster.mon_manager.run_cluster_cmd(args="osd blocklist clear")
+        except CommandFailedError:
+            # Fallback for older Ceph cluster
+            try:
+                blocklist = json.loads(self.mds_cluster.mon_manager.raw_cluster_cmd("osd",
+                                      "dump", "--format=json-pretty"))['blocklist']
+                log.info("Removing {0} blocklist entries".format(len(blocklist)))
+                for addr, blocklisted_at in blocklist.items():
+                    self.mds_cluster.mon_manager.raw_cluster_cmd("osd", "blocklist", "rm", addr)
+            except KeyError:
+                # Fallback for more older Ceph cluster
+                blacklist = json.loads(self.mds_cluster.mon_manager.raw_cluster_cmd("osd",
+                                      "dump", "--format=json-pretty"))['blacklist']
+                log.info("Removing {0} blacklist entries".format(len(blacklist)))
```

Comment:
nit: typo
**blacklist** should be changed to **blocklist**

Also, you could write log strings as **f""** format string in both **try** and **except** blocks

Reply:
> nit: typo **blacklist** should be changed to **blocklist**
> 

???

For nautilius or older cephs it should be `blacklist`. This is what I am trying to fix.

> Also, you could write log strings as **f""** format string in both **try** and **except** blocks

Just copied from the old code, I can fix this.